front/src/modules/timesStops/TimesStops.tsx

```diff
@@ -97,7 +97,6 @@ const TimesStops = ({
         } else {
           rowData.isMarginValid = true;
           if (op.fromRowIndex === 0) {
-            rowData.arrival = null;
```

Comment:
Not exactly sure what this line does, if anyone have some clarifications

Reply:
Editoast does not accept any arrival time for the first pathStep. We should handle this when we format the payload for the backend instead